shell/Cargo.toml

```diff
@@ -38,9 +38,9 @@ tezos_protocol_ipc_client = { path = "../tezos/protocol-ipc-client" }
 [dev-dependencies]
 serial_test = "0.5"
 slog-async = "2.6"
-slog-term = "2.8"
+slog-term = "2.9"
 fs_extra = "1.2.0"
-zip = "0.5.5"
+zip = { git = "https://github.com/zip-rs/zip.git" }
```

Comment:
@akoptelov it would be better to have a tag here, just in case future changes break compatibility